test/units/testsuite-68.sh

```diff
@@ -183,27 +187,43 @@ Description=TEST-68-PROPAGATE-EXIT-STATUS handle service exiting in failure (tem
 ExecStartPre=echo "triggered by %i"
 ExecStartPre=/tmp/check_on_failure.sh
 ExecStart=/tmp/check_on_failure.sh
+ExecStart=/tmp/check_on_failure.sh
+ExecStartPost=test -z '$MONITOR_SERVICE_RESULT'
 EOF
 
 systemctl daemon-reload
 
+: "-------I----------------------------------------------------"
 systemctl start testservice-failure-68.service
 wait_on_state_or_fail "testservice-failure-exit-handler-68.service" "inactive" "10"
+
+: "-------II---------------------------------------------------"
 systemctl start testservice-success-68.service
 wait_on_state_or_fail "testservice-success-exit-handler-68.service" "inactive" "10"
 
+# Let's get rid of the failed units so the tests below don't fail
+systemctl reset-failed
+
 # Test some transient units since these exit very quickly.
+: "-------III--------------------------------------------------"
 systemd-run --unit=testservice-transient-success-68 --property=OnSuccess=testservice-success-exit-handler-68.service sh -c "exit 0"
 wait_on_state_or_fail "testservice-success-exit-handler-68.service" "inactive" "10"
+
+: "-------IIII-------------------------------------------------"
```

Comment:
`IV` ?

Reply:
Apparently Romans used "IIII". "IV" is a medieval form. I thought it'd be nice to go back to the roots.

Reply:
my whole school education has been a lie